src/challenges/challengeOne/2.Dropdown/Dropdown.test.tsx

```diff
@@ -1,8 +1,35 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Dropdown } from "./Dropdown";
-
-// describe("Dropdown", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import { Dropdown } from "./Dropdown";
+
+describe("Dropdown", () => {
+  it("renders label correctly", () => {
```

Comment:
You're not actually getting the label in this test. `getByLabelText` gets the input the label is associated with, so you're really testing the input is present with the label. If you wanted the label directly you would have to use `getByText`